pkg/reconciler/openshift/tektonaddon/namespacedTask.go

```diff
@@ -0,0 +1,40 @@
+package tektonaddon
```

Comment:
nit: Please put Tekton authors header that we put in other files.

Reply:
Yep added the authors header, Thanks for pointing that out :)